### DISCUSSION TITLE
Wie kann ich mit REDAXO einen Instagram-Feed verarbeiten?

### DISCUSSION THREAD

**Discussion by tyrant88** [FriendsOfREDAXO/.github#35] [Q&A] [answered]
Ich möchte mir eine Instagram-Wall bauen, in der immer die neusten Instagram-Posts angezegt werden. 
Die Module kann ich bauen, aber wie hole ich die Posts ab? Geht das mit dem AddOn "Feeds"?

**Comment by skerbis** [comment, marked as answer]
Die aktuelle Instagram Implementierung funktioniert nicht mehr in feeds. 
Es bietet sich ein Dienst wie [rss.app an ](https://rss.app) um die Posts in einen RSS-Feed zu wandeln. 
Diesen RSS-Feed kann man mit feeds dann auslesen.
